### ISSUE TITLE
make new command !rank similar to !lvl (which works for bosses), to show boss kc+rank

### ISSUE THREAD

**Issue by CWorldEnder** [runelite/runelite#10897]
Currently _!kc Vorkath_ would show this:
`Vorkath kill count: 195`
and _!lvl Vorkath_ would show this:
`Level Vorkath: 195 Experience: -1 Rank 83,429`

This is cool, however the experience part doesn't really fit, using Level Vorkath as the prefix sounds weird, and using !lvl to show kc is unintuitive.
### Suggestion
make !kc work like !lvl and use the highscores, or introduce new command !rank which works analogous to how !lvl Vorkath would work, while removing the "Experience" part, making _!rank vorkath_ show:
`Vorkath kill count: 195 Rank 83,429`
### Pros
+ Flex your KC rank

### Cons
- Highscores not up-to-date until you relog
- Highscores unreliable recently, 
=> add new "!rank"  command instead of changing old "!kc" command

**Comment by iawhd** [comment]
This could probably just be appended onto the !kc <boss> command.

**Comment by CWorldEnder** [comment]
Yeah I was thinking of that too, but that maybe wouldn't be received as well because highscores have been unreliable lately, and you would also have to relog to show people your up-to-date kc.

Maybe if it would only use highscores for the rank? And obtaining the kc would work the same way as it does now

**Comment by Chee-RS** [comment]
> Yeah I was thinking of that too, but that maybe wouldn't be received as well because highscores have been unreliable lately, and you would also have to relog to show people your up-to-date kc.
I can think of 2 solutions, I don't know if they will work but:
1. Take your KC and somehow plug it into HS and take the lowest rank (minus one) with that KC and give that, at least until relog
2. Maybe if it can't access HS it just shows the default for now, which is just KC
